documentation/modules/migrating-virtual-machines-cli.adoc

```diff
@@ -31,19 +51,33 @@ kind: Secret
 metadata:
   name: <secret>
   namespace: {namespace}
+  ownerReferences:
```

Comment:
According to the definition of ownerReference `List of objects depended by this object. If ALL objects in the list have been deleted, this object will be garbage collected. `, this filed is for automatically removing the secret when the associated provider is removed. I think it's not a must to specify it when creating the secret.

Besides, when defining this field, it requires `uid`, otherwise it will report error `Error "Invalid value: "": uid must not be empty" for field "metadata.ownerReferences.uid".`
This `uid` is the uid of the provider, you can't get it before the provider is created.

Reply:
@qiyuann @ahadas Should I remove "ownerReference"? If I understand the 2nd paragraph of Qin's comment, even defining it as "optional" could cause trouble for the user.

Reply:
interesting, I've never defined an owner-reference from the cli (but always from the code) so I was not aware of that error when not specifying the uid, thanks @qiyuann for checking that

I wouldn't remove it but say it's optional and add the uid with value <provider uid>

Reply:
Users could modify the secret to add the owner-reference after the provider is created.

Reply:
OK, I have:
@ahadas @qiyuann 
-  added a callout that `ownerReference` is optional
- added  `uid: <provider_uid>` as the last parameter in `ownerReference`
I did not add a callout to explain `<provider_uid>` or give values for it. I don't know enough to define it; please suggest something if `<provider_uid>` is not  clear enough.

Reply:
the current state is ok